sdk/src/compute_budget.rs

```diff
@@ -41,6 +41,8 @@ pub enum ComputeBudgetInstruction {
     /// Set a compute unit price in "micro-lamports" to pay a higher transaction
     /// fee for higher transaction prioritization.
     SetComputeUnitPrice(u64),
+    /// Set a specific transaction-wide account data size limit, in bytes, is allowed to allocate.
+    SetAccountsDataSizeLimit(u32),
```

Comment:
Went back-n-forth on if better to pass TLV list as parameter, which would reduce number of IXs, but pushes complexity to data parsing. Unless 64 IXs per TX is a real and a big bottleneck, imo having less native programs but more instructions is a proven and stable option. Then again, my strong opinion loosely hold.

Reply:
Assuming my tx includes 6ixs loading the same 1MB account, is my account size limit 6MB or 1MB?

Reply:
afaik it is known that account can be loaded multiple times, this may have been fixed in loader code - i need to check. But in any case, it'd be 60MB before fix, and 10MB if addressed.

Reply:
could be dependent on runtime v2 release, if so that's going to be an important upgrade 😂